src/lib/components/multiDropzone/index.tsx

```diff
@@ -64,23 +41,35 @@ export default ({
   maxFiles = 0,
   textOverrides,
 }: Props) => {
-  const [error, setError] = useState('');
+  const [error, setError] = useState<ErrorMessage | null>();
+  const resetError = () => setError(null);
+  const formattedAccept = getFormattedAcceptObject(accept);
+  const fileList = formatAcceptFileList(formattedAccept);
+  const placeholder = `${textOverrides?.supportsTextShort || "Supports"} ${fileList || "JPEG, PNG, PDF"}`;
 
   const onDrop = useCallback(
     (acceptedFiles: File[], filesRejected: FileRejection[]) => {
-      setError('');
+      resetError();
 
       if (filesRejected.length > 0) {
-        setError(filesRejected[0].errors[0].message);
+        setError(getErrorMessage(
+          filesRejected[0].errors[0],
```

Comment:
What will happen if I drag and drop multiple files and more than one of them are not valid?
If I see correctly, this implementation will only take the error of the first rejected file.
Do you think it'd be a better idea to have an array of errors?